theano/gof/graph.py

```diff
@@ -406,6 +418,12 @@ def __str__(self):
         else:
             return "<%s>" % str(self.type)
 
+    def __get_test_value__(self):
```

Comment:
There is already such implementation in theano.gof.op.get_test_value(). Also, the name isn't very good, it make me thing it return the test value, not string representation.

I would delete this method and refactor to don't need it and call theano.gof.op.get_test_value() to be consistent in how we handle that. This would also make it print constant and shared variable value. Is that good?